openapi/data_repository_service.swagger.yaml

```diff
@@ -322,25 +322,30 @@ definitions:
   ServiceInfo:
     type: object
     required:
+      - id
+      - name
       - version
     description: >-
       Useful information about the running service.
     properties:
-      version:
+      id:
         type: string
-        description: Service version
-      title:
+        description: Unique ID of this service. Reverse domain name notation is recommended, though not required.  
+      name:
         type: string
-        description: Service name
+        description: Name of this specific service.
       description:
         type: string
-        description: Service description
-      contact:
-        type: object
-        description: Maintainer contact info
-      license:
-        type: object
-        description: License information for the exposed API
+        description: Description of the service.
+      documentationUrl:
```

Comment:
I feel it might be worth adding an explicit schemaURL to allow machine code to detect the OpenAPI/Swagger schema used by the service and self-configure. This could point to the GA4GH spec YAML or a local copy that may provide non-breaking extensions.
```yaml
schemaUrl:
  type: string
  description: URL of the schema of this service (RFC 3986 format).
````

Reply:
I like it, but I think that should be done GA4GH-wide, and we should be consistent -- maybe open an issue in https://github.com/ga4gh-discovery/ga4gh-service-info? (Not sure how they're handling enhancement requests there.)

Reply:
PR https://github.com/ga4gh-discovery/ga4gh-service-info/pull/27 added.

Reply:
Yep, that's right @dglazer, any changes folks want for this should go to https://github.com/ga4gh-discovery/ga4gh-service-info

I'll make my best attempt to keep these in sync but I suspect the powers that be will allow us to bring in last minute changes to /service-info when that finalizes and before we cut an official 1.0 on our APIs.